packages/fluentui/react-northstar/src/components/Form/FormInput.tsx

```diff
@@ -21,13 +21,18 @@ const FormInput = compose<'input', FormInputProps, FormInputStylesProps, Selecte
   {
     className: formInputClassName,
     displayName: 'FormInput',
+    handledProps: ['label', 'labelPosition', 'required'],
```

Comment:
Should we specify these in the typings + propTypes?

Reply:
Not really, it takes from the `Input`:

<img width="1057" alt="Screenshot 2020-07-10 at 21 59 07" src="https://user-images.githubusercontent.com/8545105/87193277-a68a9780-c2f8-11ea-85bc-6b6d0782f7a5.png">